openedx/core/lib/url_utils.py

```diff
@@ -13,14 +13,14 @@ def quote_slashes(text):
     ';;'. By making the escape sequence fixed length, and escaping
     identifier character ';', we are able to reverse the escaping.
     """
-    return re.sub(ur'[;/]', _quote_slashes, text)
+    return re.sub(r'[;/]', _quote_slashes, text)
 
 
 def unquote_slashes(text):
     """
     Unquote slashes quoted by `quote_slashes`
     """
-    return re.sub(r'(;;|;_)', _unquote_slashes, text)
+    return re.sub(u'(;;|;_)', _unquote_slashes, text)
```

Comment:
The str should not be changed to unicode

Reply:
Can we make the patterns in these two functions use the same kind of string?  I'd be inclined to have them both use the `r` prefix to make it more obvious that they're regex patterns, but honestly don't have a strong opinion as long as they match (which they currently don't either before or after these changes).